scripts/deployment/mainnet.json

```diff
@@ -77,25 +83,28 @@
             "l1_messenger": ["OptimismL1Messenger", "0x6D4528d192dB72E282265D6092F4B872f9Dff69e"],
             "l2_messenger": "OptimismL2Messenger",
             "finality_period": 604800,
+            "transfer_cost": "",
+            "target_weight_ppm": "",
```

Comment:
Here we use a string, elsewhere an integer `0`.

Reply:
I changed it to fictitious values for all test configs and left it to `""` for `mainnet.json`, so that we are forced to put in numbers upon deployment.